src/node_tests/ndt_client.js

```diff
@@ -214,10 +230,10 @@ function ndt_c2s_test() {
         connection_open = false,
         TRANSMITTED_BYTES = 0,
         // The NDT protocol wants 8192 bytes at a time, and with masking
-        // (required for all clients) the websocket header is 8 bytes, which
-        // means we want to have a message payload of (8192 - 8) bytes to bring
-        // the total up to precisely 8192.
-        data_to_send = new Uint8Array(8192 - 8),
+        // (required for all clients) the websocket header is 14 bytes, which
+        // means we want to have a message payload of (81920 - 14) bytes to bring
+        // the total up to precisely a multiple of 8192.
+        data_to_send = new Uint8Array(81920 - 14),
```

Comment:
Just a clarifying question -- we didn't observe this making the performance difference we expected.

If another developer were to adopt this code as an example of using NDT via JS, is the larger buffer still preferable? If so, this is fine.